pom.xml

```diff
@@ -1910,7 +1911,7 @@
         <carbon.kernel.registry.imp.pkg.version>[1.0.1, 2.0.0)</carbon.kernel.registry.imp.pkg.version>
 
         <!-- Carbon Commons version-->
-        <carbon.commons.version>4.4.0</carbon.commons.version>
+        <carbon.commons.version>4.4.1-SNAPSHOT</carbon.commons.version>
```

Comment:
You can't use snapshots (need to get a new release of the commons component with the required fix)